### PR TITLE
Dockerize mockpass

### PR DESCRIPTION
**Pull request by yong-jie**
Proposing a simple Dockerfile that allows mockpass to be containerized and run with docker-compose.

Appreciate your inputs on whether this would be appropriate.

Picking an alpine image as base due to the significantly smaller number of vulnerabilities.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:12-alpine3.9
```
Comment (LoneRifle) [inline Dockerfile]: Why not just latest-alpine?

Comment (yong-jie) [reply Dockerfile]: was thinking of LTS actually, though maybe it isnt necessary?

Comment (LoneRifle) [reply Dockerfile]: I lean towards latest only because I'm not too worried about breakages with this (heaven forbid if you use this in production)

Comment (yong-jie) [reply Dockerfile]: latest-alpine apparently doesnt exist, gonna just use latest

**Pull request merged**